eve/main.yml

```diff
@@ -927,6 +928,129 @@ stages:
+              minor=$(echo "%(prop:product_version)s" | cut -d'.' -f2) &&
+              patch=$(echo "%(prop:product_version)s" | cut -d'.' -f3) &&
+              echo "$major.$minor.$(( patch>0 ? patch-1 : 0 ))"
+      - SetPropertyFromCommand:
+          name: Set the patch version flag
+          property: patch_present
+          env:
+            PRODUCT_VERSION: "%(prop:product_version)s"
+            PRODUCT_VERSION_PREV_PATCH: "%(prop:product_version_prev_patch)s"
+          command: |-
+              if [ "$PRODUCT_VERSION" != "$PRODUCT_VERSION_PREV_PATCH" ]; then
+                 echo "true"
+              fi
+      - TriggerStages:
+          name: Trigger single-node steps with Patch Artifact ISO
+          doStepIf: "%(prop:patch_present)s"
+          stage_names:
+            - single-node-upgrade-patch-centos
+
+# --- Upgrade for PATCH versions ---
+  single-node-upgrade-patch-centos:
+    worker: *single_node_worker
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
+          <<: *add_final_status_artifact_failed
+          env:
+            <<: *_env_final_status_artifact_failed
+            STEP_NAME: single-node-upgrade-patch-centos
+      - ShellCommand: *setup_cache
+      - ShellCommand: *ssh_ip_setup
+      # --- Get ISO for version N ---
+      - ShellCommand: *retrieve_iso
+      - ShellCommand: *retrieve_iso_checksum
+      - ShellCommand: *check_iso_checksum
+      # --- Get VERSION PATCH ISO for version N-1 from Artefacts ---
+      - ShellCommand: &retrieve_prev_patch_version_iso_checksum
+          <<: *retrieve_iso_checksum
+          name: Retrieve previous PATCH ISO image checksum
+          env: &_env_previous_patch
+            DEST_DIR: "/tmp"
+            BASE_URL: "http://artifacts/builds/github:scality:metalk8s:\
+                      promoted-%(prop:product_version_prev_patch)s"
+      - ShellCommand: &retrieve_prev_patch_version_iso
+          <<: *retrieve_iso
+          name: Retrieve previous PATCH ISO image
+          env:
+            <<: *_env_previous_patch
+            MAX_ATTEMPTS: "300"
+      - ShellCommand: &check_prev_patch_version_iso_checksum
+          <<: *check_iso_checksum
+          name: Check previous PATCH ISO image with checksum
+          workdir: "/tmp"
+      # --- Prepare for Bootstrap with PATCH version N-1 ---
+      - ShellCommand:
+          <<: *create_mountpoint
+          env: &_env_prev_patch_version
+            PRODUCT_VERSION: "%(prop:product_version_prev_patch)s"
+      - ShellCommand:
+          <<: *mount_iso
+          env:
+            <<: *_env_prev_patch_version
+            ISO_PATH: /tmp/metalk8s.iso
+      - ShellCommand:
+          <<: *bootstrap_config
+          env: *_env_prev_patch_version
+      # --- Install patch version N-1 ---
+      - ShellCommand:
+          <<: *run_bootstrap
+          env: *_env_prev_patch_version
+      - ShellCommand:
+          name: Provision Prometheus and AlertManager storage
```

Comment:
Not sure this step is really mandatory since we don't test anything after patch version installation, and you already create the volumes after the upgrade.

Reply:
Yes, it is not mandatory.